packages/types/src/workflow/product-category/index.ts

```diff
@@ -14,3 +15,8 @@ export interface UpdateProductCategoryWorkflowInput {
   id: string
   data: UpdateProductCategoryDTO
 }
+
+export interface BatchUpdateProductsOnCategoryWorkflowInput
+  extends LinkMethodRequest {
+  id: string
+}
```

Comment:
There's also `LinkWorkflowInput`

```suggestion
export interface BatchUpdateProductsOnCategoryWorkflowInput
  extends LinkWorkflowInput {}
```

Reply:
Nice, will use that